runhouse/main.py

```diff
@@ -129,7 +132,7 @@ def notebook(
     c.notebook()
 
 
-@app.command()
+@cluster_app.command("ssh")
 def ssh(cluster_name: str, up: bool = typer.Option(False, help="Start the cluster")):
     """SSH into a cluster created elsewhere (so `ssh cluster` doesn't work out of the box) or not yet up."""
```

Comment:
this docstring appears in the "--help", i think we can condense it a bit
```suggestion
    """SSH into a cluster created elsewhere or not yet up."""
```